dev-docs/modules/userid-submodules/liveintent.md

```diff
@@ -27,6 +27,15 @@ There are two ways to add the functionality of LiveIntent Identity sub-module to
     LiveConnectMode=minimal gulp build --modules=liveIntentIdSystem
     ```
 
+3. The hub version, which requires a live-connect tag (>=3.0.0) installed on your page in addition to prebid.
+This will significantly reduce the size of your prebid bundle and allow you to use the newest features of live-connect. All non-hub version might be deprecated in the future.
```

Comment:
```suggestion
This will significantly reduce the size of your Prebid.js bundle and allow you to use the newest features of LiveConnect. All non-hub versions will be deprecated in the future.
```